build/util/has_starboard.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+# Copyright 2023 The Cobalt Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Script for checking if we have the starboard python modules on the path."""
+
+try:
+  import starboard # pylint: disable=unused-import
```

Comment:
Wouldn't this incorrectly pass if PYTHONPATH is set to point to the internal repo but we're building the external repo or vice versa?

Reply:
Indeed it would. I guess it should use the same check as was in `gn.py`

Reply:
I'm planning to abandon this change: https://github.com/youtube/cobalt/pull/1531/files

but it'd be good to ensure the path-checks are robust for windows quirkiness with forward and backslash

Reply:
I realized that the changes in #1531 doesn't handle a PYTHONPATH with more than one entry. I fixed that in my script and tested it with a multitude of variations.